docs/reference/sailing/miscellaneous.md

```diff
@@ -0,0 +1,95 @@
+
+A _**Bearing**_ is used to describe one point in relation to another: the Bearing of point "A" from point "B"
+is the direction you would would look towards if you wanted to see point "A" while standing at point "B". A _Range_
+is the distance between points "A" and "B", so that a Bearing and Range together can locate point "A" relative to point
+"B" in polar co-ordinates. There are two main ways of measuring bearings:
+
+- A _True Bearing_ is a bearing where the angle convention is as follows: 0° is towards the **North**,
+angles increase in the clockwise direction, and angles are typically bounded within [0°, 360°)]
+- A _Relative Bearing_ is a bearing where the angle convention is as follows: 0° is **straight forwards relative to the
+boat**, and angle measurements increase in the clockwise direction. Angles may be bounded in [-180°, 180°) or [0°, 360°)
+
+In the example below, the boat "B" has a Heading (H) of 30°. The True Bearing (Bt) of the Lighthouse "A"
+from the boat is 90°. The Relative Bearing (Br) of the lighthouse from the boat is 60°.
+
+![image](../../assets/images/sailing/bearing_vs_heading.jpg)
+
+### Track Made Good
+
+Boats do not necessarily travel in the same direction as their _**Heading**_, due to the effects of ocean current and
+wind. The direction that the boat moves in relative to the sea floor is called the _**Track Made Good**_. This is the
+same as if you measured motion compared to land or with a GPS.
+
+![image](../../assets/images/sailing/track_made_good.jpg)
+
+## True, Apparent, and Boat Wind
+
+- _True Wind_ is the wind vector (speed and direction) which you would measure while standing on land (or motionless at
+sea with unchanging GPS co-ordinates). In sailbot code, this may be referred to as _**Global Wind**_. When people
+refer to "the wind", they normally mean True Wind.
+- _Boat Wind_ is the wind vector which you would measure while standing on a moving boat when the True Wind speed is 0.
+This means that boat wind always blows straight onto the bow of the boat, and the magnitude of the boat wind is equal to
+the speed of the boat.
+- _**Apparent Wind**_ is the vector sum of the True Wind and the Boat Wind. This is the wind that you would measure while
+standing on a moving boat more generally, even if there is non-zero wind. The apparent wind is also what our wind
+sensors measure, and what our sails feel.
+
+In the example below, suppose the wind is blowing from the North at 4 m/s, and suppose the boat is moving **towards**
+the East at 3 m/s.
+
+- The True Wind everywhere is blowing at 4 m/s from the North
+- The Boat Wind onboard the boat is blowing **from** the East at 3 m/s
+- The Apparent Wind onboard the boat is has a magnitude of sqrt(3^2 + 4^2) = 5 m/s, and is coming from a true bearing
+of 36.9°.
+
+![image](../../assets/images/sailing/wind_types.jpg)
+
+## Tack
+
+In the [Types of Turn](https://ubcsailbot.github.io/docs/reference/sailing/turning)
```

Comment:
In markdown you can link to pages locally as well, for example `[Types of Turn](./turning.md)`

Reply:
Thanks, this is much better. I've made the change.